src/integrationTest/java/uk/gov/hmcts/darts/cases/service/CloseOldCasesProcessorTest.java

```diff
@@ -95,8 +120,50 @@ void givenClosedEventsUseDateAsClosedDate() {
         assertEquals(RetentionConfidenceCategoryEnum.AGED_CASE, caseRetentionEntity.getConfidenceCategory());
     }
 
+    @Test
+    void closeCases_shouldCloseCases_andUseDateAsClosedDate_andSetNullConfidenceReasonAndScore_whenNoConfidenceMappingExists() {
+        // Given createAndSaveRetentionConfidenceCategoryMappings() is not invoked, so no confidence mappings exist in the DB
+
+        // And
+        HearingEntity hearing = dartsDatabase.createHearing("a_courthouse", "1", "1078", LocalDateTime.now().minusYears(7).plusMonths(3));
+
+        OffsetDateTime closeDate = OffsetDateTime.now().minusYears(7);
+
+        EventEntity eventEntity1 = dartsDatabase.getEventStub().createEvent(hearing, 8);//Re-examination
```

Comment:
What does comment on this line mean?

Reply:
I will review the comments in this test method, I have copy/pasted an existing test setup to test a slightly different scenario here.